src/main/java/seedu/address/logic/commands/delivery/DeliveryListCommand.java

```diff
@@ -64,46 +68,86 @@ public DeliveryListCommand(DeliveryStatus status, Integer customerId, Date deliv
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
+        logger.info("Executing DeliveryListCommand: status "
+            + status + ", customerId "
+            + customerId + " deliveryDate "
+            + deliveryDate + " and sortType "
+            + sortType);
         // User cannot perform this operation before logging in
         if (!model.getUserLoginStatus()) {
```

Comment:
Would you wanna add a logger.warning here for each separate path? Can log more info that way. I saw @zhonghan721 do this.